CORS error for light client on all transport servers on mainnet

## Description
The light client encounters CORS exceptions while calling the mainnet matrix server API's at startup
This is not the problem of the light client but its just that the matrix servers or for that matter any other servers if they are called from a different domain and return the reply without the `Access-Control-Allow-Origin` headers set then the browser blocks the response. 

![light_client_cors_error](https://user-images.githubusercontent.com/15123108/88906822-86098980-d275-11ea-86ae-29fc53ee5e69.png)


## Acceptance criteria
- The only thing we can do in this case is to request RSB providers to allow certain domains like 
1. https://lightclient.raiden.network/
2. http://localhost:8080/
may be some more commonly used domains used by light-client users as allowed on the CORS headers 
## Tasks
-  [ ]  RSB providers to Send CORS headers (Access-Control-Allow-Origin)

[raiden_0x14efA2b271969A46A094A23e0d49E32C1B617f89_2020-07-30T07_39_20.313Z.log](https://github.com/raiden-network/light-client/files/5000313/raiden_0x14efA2b271969A46A094A23e0d49E32C1B617f89_2020-07-30T07_39_20.313Z.log)


@christianbrb please check whether the content of this issue is correct.

One observation when i used the `https://transport.raiden.badgateway.tech` server while transferring I got 

```
index.js:218 GET https://transport.raiden.badgateway.tech/_matrix/client/r0/presence/%400xf128a427deff3a4f792373f4be38beecd07d4ffe%3Atransport.mainnet.raiden.anyblock.tools/status?access_token=MDAyZWxvY2F0aW9uIHRyYW5zcG9ydC5yYWlkZW4uYmFkZ2F0ZXdheS50ZWNoCjAwMTNpZGVudGlmaWVyIGtleQowMDEwY2lkIGdlbiA9IDEKMDA1ZmNpZCB1c2VyX2lkID0gQDB4MTRlZmEyYjI3MTk2OWE0NmEwOTRhMjNlMGQ0OWUzMmMxYjYxN2Y4OTp0cmFuc3BvcnQucmFpZGVuLmJhZGdhdGV3YXkudGVjaAowMDE2Y2lkIHR5cGUgPSBhY2Nlc3MKMDAyMWNpZCBub25jZSA9IDBacmJCPUVsN2JDJl5YckwKMDAyZnNpZ25hdHVyZSBM9K_cH1Pav5wc0vNsB6DP5MvAF-ukfVRgEMwQwBhoMAo 403
run_xhr @ index.js:218
request @ index.js:179
eval @ browser-index.js:53
eval @ polyfills.ts:19
_request @ http-api.js:716
requestOtherUrl @ http-api.js:592
request @ http-api.js:550
authedRequest @ http-api.js:502
getUserPresence @ matrix.ts:32
eval @ presence.ts:77
MergeMapSubscriber._tryNext @ mergeMap.ts:139
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToIterable.ts:21
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToPromise.ts:10
Promise.then (async)
eval @ subscribeToPromise.ts:8
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
eval @ defer.ts:66
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
FilterOperator.call @ filter.ts:72
Observable.subscribe @ Observable.ts:215
ScanOperator.call @ scan.ts:77
Observable.subscribe @ Observable.ts:215
TakeLastOperator.call @ takeLast.ts:69
Observable.subscribe @ Observable.ts:215
DefaultIfEmptyOperator.call @ defaultIfEmpty.ts:59
Observable.subscribe @ Observable.ts:215
MapOperator.call @ map.ts:60
Observable.subscribe @ Observable.ts:215
MapOperator.call @ map.ts:60
Observable.subscribe @ Observable.ts:215
CatchOperator.call @ catchError.ts:108
Observable.subscribe @ Observable.ts:215
subscribeToResult @ subscribeToResult.ts:37
ExhaustMapSubscriber._innerSub @ exhaustMap.ts:132
ExhaustMapSubscriber.tryNext @ exhaustMap.ts:122
ExhaustMapSubscriber._next @ exhaustMap.ts:108
Subscriber.next @ Subscriber.ts:100
WithLatestFromSubscriber._next @ withLatestFrom.ts:142
Subscriber.next @ Subscriber.ts:100
Subject.next @ Subject.ts:71
GroupBySubscriber._group @ groupBy.ts:203
GroupBySubscriber._next @ groupBy.ts:162
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
MapSubscriber._next @ map.ts:90
Subscriber.next @ Subscriber.ts:100
Subject.next @ Subject.ts:71
AsyncSubject.complete @ AsyncSubject.ts:46
eval @ init.ts:128
TapSubscriber._next @ tap.ts:121
Subscriber.next @ Subscriber.ts:100
TakeSubscriber._next @ take.ts:93
Subscriber.next @ Subscriber.ts:100
FilterSubscriber._next @ filter.ts:102
Subscriber.next @ Subscriber.ts:100
TakeWhileSubscriber.nextOrComplete @ takeWhile.ts:104
TakeWhileSubscriber._next @ takeWhile.ts:100
Subscriber.next @ Subscriber.ts:100
Notification.observe @ Notification.ts:48
ObserveOnSubscriber.dispatch @ observeOn.ts:103
AsyncAction._execute @ AsyncAction.ts:128
QueueAction.execute @ QueueAction.ts:34
AsyncScheduler.flush @ AsyncScheduler.ts:65
QueueAction.schedule @ QueueAction.ts:28
Scheduler.schedule @ Scheduler.ts:68
AsyncScheduler.schedule @ AsyncScheduler.ts:49
ObserveOnSubscriber.scheduleMessage @ observeOn.ts:109
ObserveOnSubscriber._next @ observeOn.ts:113
Subscriber.next @ Subscriber.ts:100
Subscriber._next @ Subscriber.ts:140
Subscriber.next @ Subscriber.ts:100
Subscriber._next @ Subscriber.ts:140
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToArray.ts:11
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToArray.ts:11
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
Subscriber._next @ Subscriber.ts:140
Subscriber.next @ Subscriber.ts:100
TapSubscriber._next @ tap.ts:126
Subscriber.next @ Subscriber.ts:100
ThrowIfEmptySubscriber._next @ throwIfEmpty.ts:65
Subscriber.next @ Subscriber.ts:100
TakeSubscriber._next @ take.ts:93
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
Subscriber._next @ Subscriber.ts:140
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
MapToSubscriber._next @ mapTo.ts:71
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToPromise.ts:10
Promise.then (async)
eval @ subscribeToPromise.ts:8
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToArray.ts:11
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
MapToOperator.call @ mapTo.ts:52
Observable.subscribe @ Observable.ts:215
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
MapSubscriber._next @ map.ts:90
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToPromise.ts:10
Promise.then (async)
eval @ subscribeToPromise.ts:8
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
MapOperator.call @ map.ts:60
Observable.subscribe @ Observable.ts:215
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
MergeMapSubscriber.notifyNext @ mergeMap.ts:174
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
Subscriber._next @ Subscriber.ts:140
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToPromise.ts:10
Promise.then (async)
eval @ subscribeToPromise.ts:8
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
CatchOperator.call @ catchError.ts:108
Observable.subscribe @ Observable.ts:215
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToPromise.ts:10
Promise.then (async)
eval @ subscribeToPromise.ts:8
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
eval @ defer.ts:66
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
CatchOperator.call @ catchError.ts:108
Observable.subscribe @ Observable.ts:215
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
Subscriber._next @ Subscriber.ts:140
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToArray.ts:11
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
eval @ defer.ts:66
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
CatchOperator.call @ catchError.ts:108
Observable.subscribe @ Observable.ts:215
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
TakeOperator.call @ take.ts:73
Observable.subscribe @ Observable.ts:215
ThrowIfEmptyOperator.call @ throwIfEmpty.ts:50
Observable.subscribe @ Observable.ts:215
DoOperator.call @ tap.ts:82
Observable.subscribe @ Observable.ts:215
RetryWhenOperator.call @ retryWhen.ts:38
Observable.subscribe @ Observable.ts:215
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
ThrowIfEmptySubscriber._next @ throwIfEmpty.ts:65
Subscriber.next @ Subscriber.ts:100
TakeSubscriber._next @ take.ts:93
Subscriber.next @ Subscriber.ts:100
CombineLatestSubscriber.notifyNext @ combineLatest.ts:314
InnerSubscriber._next @ InnerSubscriber.ts:18
Subscriber.next @ Subscriber.ts:100
DistinctUntilChangedSubscriber._next @ distinctUntilChanged.ts:122
Subscriber.next @ Subscriber.ts:100
MapSubscriber._next @ map.ts:90
Subscriber.next @ Subscriber.ts:100
ReplaySubject._subscribe @ ReplaySubject.ts:84
Observable._trySubscribe @ Observable.ts:239
Subject._trySubscribe @ Subject.ts:117
Observable.subscribe @ Observable.ts:221
MapOperator.call @ map.ts:60
Observable.subscribe @ Observable.ts:215
DistinctUntilChangedOperator.call @ distinctUntilChanged.ts:75
Observable.subscribe @ Observable.ts:215
subscribeToResult @ subscribeToResult.ts:37
CombineLatestSubscriber._complete @ combineLatest.ts:289
Subscriber.complete @ Subscriber.ts:127
eval @ subscribeToArray.ts:12
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
CombineLatestOperator.call @ combineLatest.ts:255
Observable.subscribe @ Observable.ts:215
TakeOperator.call @ take.ts:73
Observable.subscribe @ Observable.ts:215
ThrowIfEmptyOperator.call @ throwIfEmpty.ts:50
Observable.subscribe @ Observable.ts:215
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
eval @ subscribeToArray.ts:11
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
MergeMapOperator.call @ mergeMap.ts:109
Observable.subscribe @ Observable.ts:215
CatchOperator.call @ catchError.ts:108
Observable.subscribe @ Observable.ts:215
TakeUntilOperator.call @ takeUntil.ts:65
Observable.subscribe @ Observable.ts:215
eval @ using.ts:56
Observable._trySubscribe @ Observable.ts:239
Observable.subscribe @ Observable.ts:221
SubscribeOnObservable.dispatch @ SubscribeOnObservable.ts:51
AsyncAction._execute @ AsyncAction.ts:128
QueueAction.execute @ QueueAction.ts:34
AsyncScheduler.flush @ AsyncScheduler.ts:65
QueueAction.schedule @ QueueAction.ts:28
Scheduler.schedule @ Scheduler.ts:68
AsyncScheduler.schedule @ AsyncScheduler.ts:49
SubscribeOnObservable._subscribe @ SubscribeOnObservable.ts:52
Observable.subscribe @ Observable.ts:219
SubscribeOnOperator.call @ subscribeOn.ts:64
Observable.subscribe @ Observable.ts:215
ObserveOnOperator.call @ observeOn.ts:80
Observable.subscribe @ Observable.ts:215
subscribeToResult @ subscribeToResult.ts:37
MergeMapSubscriber._innerSub @ mergeMap.ts:157
MergeMapSubscriber._tryNext @ mergeMap.ts:150
MergeMapSubscriber._next @ mergeMap.ts:130
Subscriber.next @ Subscriber.ts:100
MapSubscriber._next @ map.ts:90
Subscriber.next @ Subscriber.ts:100
Subject.next @ Subject.ts:71
epicMiddleware.run @ createEpicMiddleware.js:70
start @ raiden.ts:415
_callee6$ @ raiden-service.ts:253
tryCatch @ runtime.js:63
invoke @ runtime.js:293
eval @ runtime.js:118
asyncGeneratorStep @ raiden-service.ts:66
_next @ raiden-service.ts:68
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:66
_next @ raiden-service.ts:68
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:66
_next @ raiden-service.ts:68
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:66
_next @ raiden-service.ts:68
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:66
_next @ raiden-service.ts:68
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:66
_next @ raiden-service.ts:68
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:66
_next @ raiden-service.ts:68
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:66
_next @ raiden-service.ts:68
eval @ raiden-service.ts:68
eval @ raiden-service.ts:68
connect @ raiden-service.ts:279
_callee2$ @ Home.vue:131
tryCatch @ runtime.js:63
invoke @ runtime.js:293
eval @ runtime.js:118
asyncGeneratorStep @ cjs.js?!../node_modules/.pnpm/babel-loader@8.1.0_13c776b87d2822e76e5e552d0872621e/node_modules/babel-loader/lib/index.js!../node_modules/.pnpm/ts-loader@6.2.2_typescript@3.9.7/node_modules/ts-loader/index.js?!../node_modules/.pnpm/cache-loader@4.1.0_webpack@4.43.0/node_modules/cache-loader/dist/cjs.js?!../node_modules/.pnpm/vue-loader@15.9.3_19eca3d246a5abd3297fc67c463bf5f8/node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:24
_next @ cjs.js?!../node_modules/.pnpm/babel-loader@8.1.0_13c776b87d2822e76e5e552d0872621e/node_modules/babel-loader/lib/index.js!../node_modules/.pnpm/ts-loader@6.2.2_typescript@3.9.7/node_modules/ts-loader/index.js?!../node_modules/.pnpm/cache-loader@4.1.0_webpack@4.43.0/node_modules/cache-loader/dist/cjs.js?!../node_modules/.pnpm/vue-loader@15.9.3_19eca3d246a5abd3297fc67c463bf5f8/node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:26
eval @ cjs.js?!../node_modules/.pnpm/babel-loader@8.1.0_13c776b87d2822e76e5e552d0872621e/node_modules/babel-loader/lib/index.js!../node_modules/.pnpm/ts-loader@6.2.2_typescript@3.9.7/node_modules/ts-loader/index.js?!../node_modules/.pnpm/cache-loader@4.1.0_webpack@4.43.0/node_modules/cache-loader/dist/cjs.js?!../node_modules/.pnpm/vue-loader@15.9.3_19eca3d246a5abd3297fc67c463bf5f8/node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:26
eval @ cjs.js?!../node_modules/.pnpm/babel-loader@8.1.0_13c776b87d2822e76e5e552d0872621e/node_modules/babel-loader/lib/index.js!../node_modules/.pnpm/ts-loader@6.2.2_typescript@3.9.7/node_modules/ts-loader/index.js?!../node_modules/.pnpm/cache-loader@4.1.0_webpack@4.43.0/node_modules/cache-loader/dist/cjs.js?!../node_modules/.pnpm/vue-loader@15.9.3_19eca3d246a5abd3297fc67c463bf5f8/node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:26
connect @ Home.vue:90
invokeWithErrorHandling @ vue.runtime.esm.js:1854
invoker @ vue.runtime.esm.js:2179
invokeWithErrorHandling @ vue.runtime.esm.js:1854
Vue.$emit @ vue.runtime.esm.js:3888
emit @ vue-property-decorator.js:231
emitter @ vue-property-decorator.js:257
click @ generated-ActionButton.vue?329d:35
invokeWithErrorHandling @ vue.runtime.esm.js:1854
invoker @ vue.runtime.esm.js:2179
invokeWithErrorHandling @ vue.runtime.esm.js:1854
Vue.$emit @ vue.runtime.esm.js:3888
click @ VBtn.ts:158
invokeWithErrorHandling @ vue.runtime.esm.js:1854
invoker @ vue.runtime.esm.js:2179
original._wrapper @ vue.runtime.esm.js:6917
Show 237 more frames
logger.js:49 Error fetching user presence, ignoring: M_FORBIDDEN: You are not allowed to see their presence.
```
**Error fetching user presence, ignoring: M_FORBIDDEN: You are not allowed to see their presence.**

But then somehow it did not complain on the browser and got the presence from somewhere

and while making the actual transfer it complained about CORS

```
Access to fetch at 'https://pfs.raiden.overdoze.se/api/v1/info' from origin 'http://localhost:8080' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

--- a/raiden-dapp/.env.development
+++ b/raiden-dapp/.env.development
@@ -1,4 +1,4 @@
 VUE_APP_PFS=https://pfs.demo001.env.raiden.network
-VUE_APP_TRANSPORT=https://transport.demo001.env.raiden.network
+VUE_APP_MATRIX_SERVER=https://transport.demo001.env.raiden.network
 VUE_APP_HUB=hub.raiden.eth
 VUE_APP_ALLOW_MAINNET=true

--- a/raiden-dapp/.env.e2e
+++ b/raiden-dapp/.env.e2e
@@ -2,6 +2,6 @@ VUE_APP_CONFIGURATION_URL=./e2e.json
 VUE_APP_DEPLOYMENT_INFO=./deployment_private_net.json
 VUE_APP_DEPLOYMENT_SERVICES_INFO=./deployment_services_private_net.json
 VUE_APP_PFS=http://localhost:6000
-VUE_APP_TRANSPORT=http://localhost
+VUE_APP_MATRIX_SERVER=http://localhost
 NODE_ENV=development
 E2E=true

--- a/raiden-dapp/.env.ghpages
+++ b/raiden-dapp/.env.ghpages
@@ -1,5 +1,5 @@
 NODE_ENV=production
 VUE_APP_PFS=https://pfs.demo001.env.raiden.network
-VUE_APP_TRANSPORT=https://transport.demo001.env.raiden.network
+VUE_APP_MATRIX_SERVER=https://transport.demo001.env.raiden.network
 VUE_APP_HUB=hub.raiden.eth
 VUE_APP_ALLOW_MAINNET=false

--- a/raiden-dapp/CHANGELOG.md
+++ b/raiden-dapp/CHANGELOG.md
@@ -8,6 +8,7 @@
 
 - [#1786] Introduces snackbar display for notifications
 - [#1824] Listen to channel settle events and push notifications for them
+- [#2002] Add support to VUE_APP_MATRIX_LIST_URL transpile-time env var
 
 ### Changed
 - [#1931] dApp always uses hash mode on router
@@ -23,6 +24,7 @@
 [#1786]: https://github.com/raiden-network/light-client/issues/1786
 [#1824]: https://github.com/raiden-network/light-client/issues/1824
 [#1875]: https://github.com/raiden-network/light-client/issues/1875
+[#2002]: https://github.com/raiden-network/light-client/issues/2002
 
 ## [0.10.0] - 2020-07-13
 

--- a/raiden-dapp/src/services/raiden-service.ts
+++ b/raiden-dapp/src/services/raiden-service.ts
@@ -49,7 +49,8 @@ export default class RaidenService {
         {
           pfsSafetyMargin: 1.1,
           pfs: process.env.VUE_APP_PFS,
-          matrixServer: process.env.VUE_APP_TRANSPORT,
+          matrixServer: process.env.VUE_APP_MATRIX_SERVER,
+          matrixServerLookup: process.env.VUE_APP_MATRIX_LIST_URL,
         },
         subkey
       );

